### PR TITLE
#678 Better dev experience for Kafka Streams

### PR DESCRIPTION
**Pull request by gunnarmorling**
Hey @geoand, some quick fixes while preparing our session late today :)

### CODE REVIEW

**Review by geoand — APPROVED** [review]
Lovely!

File: kafka-streams-quickstart/docker-compose.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: '3.5'
 
 services:
   zookeeper:
-    image: strimzi/kafka:0.11.3-kafka-2.1.0
+    image: strimzi/kafka:0.19.0-kafka-2.5.0
```
Comment (cescoffier) [inline kafka-streams-quickstart/docker-compose.yaml:5]: We should update all our docker-compose files.

**Review by cescoffier — APPROVED** [review]

File: kafka-streams-quickstart/docker-compose.yaml (hunk 1 above)
Comment (gunnarmorling) [reply kafka-streams-quickstart/docker-compose.yaml:5]: I couldn't find any other references to "kafka:0.11.3-kafka-2.1.0" in the quickstart repo? DYM elsewhere? I'll send a PR for updating some references I found in docs.

**Comment by gunnarmorling** [comment]
Sent https://github.com/quarkusio/quarkus/pull/12543 for the version update in the docs in the main repo. Let me know if anything else is needed to wrap this up.

File: kafka-streams-quickstart/docker-compose.yaml (hunk 1 above)
Comment (cescoffier) [reply kafka-streams-quickstart/docker-compose.yaml:5]: I was thinking about:
* https://github.com/quarkusio/quarkus-quickstarts/blob/development/kafka-quickstart/docker-compose.yaml
* https://github.com/quarkusio/quarkus-quickstarts/blob/development/kafka-panache-quickstart/docker-compose.yaml

But no worries, I will open a PR about these.

**Comment by cescoffier** [comment]
Waiting for CI, but it's good to go in! Thanks again @gunnarmorling !

**Pull request merged**